### ISSUE TITLE
How to use GPU for inference

### ISSUE THREAD

**Issue by Unkrible** [wiktorlazarski/head-segmentation#27]
Hello, thanks for the open-source of awesome work!
However, I have the question that as guided in the `Inference by HumanHeadSegmentationPipeline .predict`, GPU can not be used to accelerate the inference process.
Thus, I suggestion to add `self.device = torch.device('cuda') if torch.cuda.is_available() else torch.device('cpu')` in `class HumanHeadSegmentationPipeline`, and then modify the function `predict(self, image)` as `mdl_out = self._model(preprocessed_image.to(self.device)).detach().cpu()`

**Comment by wiktorlazarski** [comment]
Feel free to make a PR with all proposed changes 😉.

**Comment by wiktorlazarski** [comment]
Hey @9527-csroad, 

thanks for having a look at it 🙌. Amazing work 🚀! I love a time benchmarking. Also, huge shoutout for the PR https://github.com/wiktorlazarski/head-segmentation/pull/30. I promise I'll have a look at it later today.
